.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Take a look at https://docs.github.com/es/github/creating-cloning-and-archiving-repositories/about-code-owners
+*   
```

Comment:
Should we add this empty file?

Reply:
I don't know since this file describe the code owners for the repo, I don't know who should be listed in this file or if there is none remove it.